api/src/test/java/org/openmrs/api/db/PatientDAOTest.java

```diff
@@ -755,6 +755,42 @@ public void getPatients_shouldNotMatchVoidedPatientNames_SignatureNo1() {
 		Assert.assertEquals(0, patients.size());
 	}
 	
+	@Test
+	public void getPatients_shouldNotMatchVoidedPatientsWhenVoidedFalseIsPassed() {
+		List<Patient> patients = dao.getPatients("Oloo", 0, 11);
```

Comment:
What does 0 in the second parameter mean?

Reply:
@dkayiwa its an integer parameter representing the starting position and in this case the specified number is 0

Reply:
@gitcliff Are you still interested in working on this? If so, this should be updated to:

```suggestion
		List<Patient> patients = dao.getPatients("Oloo", false, 0, 11);
```